lib/cn/csched_sp3_work.c

```diff
@@ -908,7 +925,7 @@ sp3_work(
 
         switch (wtype) {
         case wtype_root:
-            n_kvsets = sp3_work_wtype_root(spn, thresh, &mark, &action, &rule);
+            n_kvsets = sp3_work_wtype_root(spn, thresh, tree->ct_route_map, &mark, &action, &rule);
```

Comment:
Put that new arg at the end of the func so that all the wtype funcs have the same signature up to this one off...

Reply:
So, turns out you don't need to pass the route map in, just get it from tn->tn_tree->ct_route_map.

Reply:
Ah, nice. Fixed it.